pkg/appmgmt/general/general.go

```diff
@@ -342,6 +343,7 @@ func (os *Manager) ListApplications() (map[string]interfaces.ApplicationMetadata
 			}
 		}
 	}
+	log.Logger().Info("Listing recoverable apps finished", zap.Int("nr of recoverable apps", len(existingApps)))
```

Comment:
"Listing recoverable apps finished" --> "application recovery statistics"
adding the two values mentioned above to the number of apps